tsconfig.json

```diff
@@ -18,10 +18,6 @@
     "sourceMap": true,
     "noFallthroughCasesInSwitch": true,
     "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
-    "paths": {
-      "*": [ "src/*", "src/library/*" ]
```

Comment:
let's add an alias for the library directory so that instead of ../.../../../../[path] we can write @lib/[path]